app/Jobs/ApplicationDeploymentJob.php

```
@@ -670,13 +670,20 @@ private function deploy_docker_compose_buildpack()
                 $build_command = "DOCKER_BUILDKIT=1 {$build_command}";
             }
 
-            // Append build arguments if not using build secrets (matching default behavior)
+            // Inject build arguments after build subcommand if not using build secrets
             if (! $this->application->settings->use_build_secrets && $this->build_args instanceof \Illuminate\Support\Collection && $this->build_args->isNotEmpty()) {
                 $build_args_string = $this->build_args->implode(' ');
```

PHPMD (2.15.0)

The variable $build_args_string is not named in camelCase. (undefined)

(CamelCaseVariableName)